src/test/scala/ai/privado/languageEngine/go/tagger/collection/CollectionFuncTaggerTest.scala

```diff
@@ -0,0 +1,164 @@
+package ai.privado.languageEngine.go.tagger.collection
+
+import ai.privado.cache.{RuleCache, TaggerCache}
+import ai.privado.model.{CatLevelOne, ConfigAndRules, Language, NodeType, RuleInfo}
+import better.files.File
+import io.joern.gosrc2cpg.{Config, GoSrc2Cpg}
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import io.joern.x2cpg.X2Cpg.applyDefaultOverlays
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.semanticcpg.language.*
+
+import scala.collection.mutable
+
+abstract class CollectionTaggerTestBase extends AnyWordSpec with Matchers with BeforeAndAfterAll {
```

Comment:
can we create separate file for `CollectionTaggerTestBase`?

Reply:
done.